src/runtime/Types/ClassObject.cs

```diff
@@ -48,6 +48,43 @@ internal NewReference GetDocString()
         }
 
 
+        /// <summary>
+        /// given an enum, write a __repr__ string formatted in the same
+        /// way as a python repr string. Something like:
+        ///   '&lt;Color.GREEN: 2&gt;';
+        /// with a binary value for [Flags] enums
+        /// </summary>
+        /// <param name="inst">Instace of the enum object</param>
+        /// <returns></returns>
+        private static string getEnumReprString(object inst)
+        {
+            var obType = inst.GetType();
+
+            var isFlags = obType.IsFlagsEnum();
+            var intValue = Convert.ToInt32(inst);
```

Comment:
This will fail for `Int64` and `UInt64`-based enums. Also you might have to separately handle `UInt64` case.